indico/modules/events/timetable/views/__init__.py

```diff
@@ -132,10 +132,14 @@ def inject_meeting_body(event, **kwargs):
     days = [(day, list(e)) for day, e in groupby(entries, lambda e: e.start_dt.astimezone(event_tz).date())]
     theme_id = view if view and view in theme_settings.themes else event.theme
     theme = theme_settings.themes[theme_id]
-    tt_tpl = theme.get('tt_template', theme['template'])
+    plugin = theme.get('plugin')
+    tpl_name = theme.get('tt_template', theme['template'])
+    tt_tpl = (plugin.name + tpl_name
+              if plugin and tpl_name[0] == ':'
```

Comment:
The wrapping per se doesn't make it clear, IMHO, but anyway, not very important.